engines/bastion_katello/app/assets/javascripts/bastion_katello/products/products.routes.js

```diff
@@ -42,13 +42,14 @@ angular.module('Bastion.products').config(['$stateProvider', function ($statePro
     });
 
     $stateProvider.state("product-discovery", {
-        abstract: true,
         url: '/products/discovery',
-        controller: 'DiscoveryController',
-        templateUrl: 'products/discovery/views/discovery-base.html'
+        templateUrl: 'products/discovery/views/discovery-base.html',
```

Comment:
Small adjustment here now makes /products/discovery redirect to /products/discovery/scan which avoids the UI trying to display an nonexistent product called 'discovery'